src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -902,9 +897,9 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
                         if (rNumericArgs(2) >= 0.0) {
```

Comment:
Suggest you test if the field is blank first. That feels safer.

Reply:
This can wait for a later PR which plans to consolidate the multiple nearly identical switch case blocks for design flow method in a common function.